packages/notebook-extension/src/index.ts

```diff
@@ -1718,7 +1718,8 @@ function addCommands(
     isEnabled
   });
   commands.addCommand(CommandIDs.restartClear, {
-    label: trans.__('Restart Kernel and Clear All Outputs…'),
+    label: trans.__('Restart Kernel and Clear All Cell Outputs…'),
```

Comment:
Is there any special meaning for the three dots at the end?

Reply:
Good point! I think that three dots make sense for "Change Kernel..." because it indicates that a pop-up dialog with more options will be shown (and for "Go to line..." and for "Find..." as all these require additional user action), but it indeed does not seem necessary here (or for any other "Restart Kernel" option).

I think we should document that ellipsis should be only added when there is a follow up choice/action need from the user.

Reply:
Added a section to the style guide on command names.